app/api/crud.py

```diff
@@ -85,16 +89,17 @@ async def get_terms(data_element_URI: str):
     cross_node_results = []
     params = {data_element_URI: data_element_URI}
 
-    for node_url in util.parse_nodes_as_list(util.NEUROBAGEL_NODES):
+    for node_url in util.parse_nodes_as_dict(util.NEUROBAGEL_NODES).keys():
         response = util.send_get_request(
             node_url + "attributes/" + data_element_URI, params
         )
 
         cross_node_results.append(response)
 
-    unique_terms = set(
-        term
-        for list_of_terms in cross_node_results
-        for term in list_of_terms[data_element_URI]
-    )
-    return {data_element_URI: list(unique_terms)}
+    unique_terms_dict = {}
+
+    for list_of_terms in cross_node_results:
+        for term in list_of_terms[data_element_URI]:
+            unique_terms_dict[term["TermURL"]] = term
```

Comment:
this works, but should we not do
```suggestion
            unique_terms_dict[term["TermURL"]] = term["Label"]
```
or whatever the key for the string value is? in other words: what's the expected shape of the returned list?

Reply:
This is a workaround to essentially remove duplicate keys or rather overwrite them.

Reply:
I get that, but why do you have to have `TermURL` be both in the key and then again in the value?

Reply:
I see. it's supposed to help structure the output at the end to resemble what we have in the node API.